.github/workflows/perf_sysbench.yml

```diff
@@ -44,6 +44,12 @@ jobs:
       - name: Add tarantool path to $PATH
```

Comment:
What is `Prev(rps)` in the table in the following cases?

* First push to a new branch, the patchset (several commits) is based on `master`.
* Same, but the commit is based on `master~N`.
* Append-only (non-force) push to the existing branch (several commits).
* Force-push to the existing branch (several commits).

Reply:
> First push to a new branch, the patchset (several commits) is based on `master`.

For now, `Prev` will be `None`

> Same, but the commit is based on `master~N`.

The same.

> Append-only (non-force) push to the existing branch (several commits).

The latest commit in the branch before merging patch set.

> Force-push to the existing branch (several commits).

We don't force-push to the 'master' and release branches.

Please note, for now, this workflow is run only on push to the 'master' and release branches, not on push to every branch.

Reply:
> Please note, for now, this workflow is run only on push to the 'master' and release branches, not on push to every branch.

I forgot that, sorry.